hs_access_control/models.py

```diff
@@ -231,12 +342,16 @@ def create_group(self, title):
         owner.
         """
         if __debug__: 
-            assert isinstance(title, basestring) 
+            assert isinstance(title, basestring)
+            assert isinstance(description, basestring)
+            if purpose:
+                assert isinstance(purpose, basestring)
```

Comment:
Just to clarify. These assert statements are only invoked during debugging and catch rather serious usage errors. The whole access control system has them and the usage here is consistent. If we are going to change this, we would do it for the whole access control system at once. That is rather easy and won't change any function. 

I left them in with the intent of deleting them once we are finished with feature development. 
Since it is rather trivial to remove them, I think they should stay temporarily. 

Actually, this approach is a significant modification from the approach that Calvin originally complained about. The previous approach had assertions that were always tested, even outside of debugging.